src/web/src/app/space/invitation-list/component.ts

```diff
@@ -200,8 +199,8 @@ export class SpaceInvitationListComponent implements OnInit {
 	}
 
 	changeRole(value, item: Detail) {
-		if (item.type == INVITATION_TYPE.WE_INVITED) {
-			this.space.changeRoleInInvitation(item.id, value)
+		if (item.type == INVITATION_TYPE.WE_INVITED || item.type == INVITATION_TYPE.WANTS_TO_JOIN) {
+			this.space.changeRoleInInvitation(String(item.id), value)
```

Comment:
Зачем тут такое преобразование? Давай так.
```suggestion
			this.space.changeRoleInInvitation(item.id, value)
```

Reply:
Готово.

Reply:
Чот не готово. ((